.github/workflows/coverage.yml

```diff
@@ -3,7 +3,11 @@ name: Coverage
 on:
   push:
     branches:
+      - dev
       - stable
+    paths:
+      - '**.rs'
+      - '**.toml'
```

Comment:
```suggestion
      - '**.toml'
      - '.github/workflows/coverage.yml'
```

You might want this workflow to run when this file is changed too

Reply:
Good idea 👍